### PR TITLE
chore: bring back lerna-release-action

### PR DESCRIPTION
**Pull request by mvayngrib**
it has passed security review

### CODE REVIEW

File: .github/workflows/version.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        required: false
+
+jobs:
+  version:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - uses: ExodusMovement/actions/setup/lerna@06f077a9081a1e973586dde8870e7116873b9b84
```
Comment (mvayngrib) [inline .github/workflows/version.yaml:33]: @sparten11740 should we use the latest here?

Comment (sparten11740) [reply .github/workflows/version.yaml:33]: yes

File: .github/workflows/publish.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        run: yarn install --immutable
+      - name: Build
+        run: yarn build
+      - uses: ExodusMovement/lerna-release-action/publish@5bc3d09d8259df70e96766ae300b8d33aa0eb08d
+        id: publish
+        with:
+          github-token: ${{ secrets.GH_AUTOMATION_PAT }}
+      - uses: ExodusMovement/athena-action@master
+        with:
+          github-token: ${{ secrets.GH_AUTOMATION_PAT }}
```
Comment (mvayngrib) [inline .github/workflows/publish.yaml:29]: ah wait, removing this one for now

**Review by 633kh4ck — APPROVED** [review]

**Review by sparten11740 — APPROVED** [review]
utACK

**Pull request merged**